tensorflow/core/kernels/image/adjust_contrast_op.cc

```diff
@@ -387,9 +387,14 @@ class AdjustContrastOpv2<CPUDevice, float> : public AdjustContrastOpV2Base {
   }
 };
 
-REGISTER_KERNEL_BUILDER(
-    Name("AdjustContrastv2").Device(DEVICE_CPU).TypeConstraint<float>("T"),
-    AdjustContrastOpv2<CPUDevice, float>);
+#define REGISTER_KERNEL(T)                                                 \
+  REGISTER_KERNEL_BUILDER(                                                 \
+      Name("AdjustContrastv2").Device(DEVICE_CPU).TypeConstraint<T>("T"),  \
+      AdjustContrastOpv2<CPUDevice, T>);
```

Comment:
nit: the trailing semicolon here leads to double semicolons when you call
```
REGISTER_KERNEL(...);
```
below.  You should remove it from here.

Reply:
Changes done as suggested to avoid double semicolons.